controlplane/pkg/apis/registry/registry.proto

```diff
@@ -64,4 +64,5 @@ service NetworkServiceRegistry {
 
 service NetworkServiceDiscovery {
     rpc FindNetworkService (FindNetworkServiceRequest) returns (FindNetworkServiceResponse);
+    rpc UpdateNSM (NetworkServiceManager) returns (NetworkServiceManager);
```

Comment:
Why is this in the NetworkServiceDiscovery rather than NetworkServiceRegistration above?

Reply:
Ah... I think I see... because its not something that you would want the client from the NSE using.

Perhaps UpdateNSM should be in its own 'service' ?